common/djangoapps/util/views.py

```diff
@@ -470,23 +273,18 @@ def build_error_response(status_code, field, err_msg):
             success = False
 
     else:
-        if not settings.ZENDESK_URL or not settings.ZENDESK_USER or not settings.ZENDESK_API_KEY:
-            raise Exception("Zendesk enabled but not configured")
+        if not settings.HELPDESK_URL or not settings.HELPDESK_USER or not settings.HELPDESK_API_KEY:
+            raise Exception("Helpdesk enabled but not configured")
 
-        custom_fields = None
-        if settings.ZENDESK_CUSTOM_FIELDS:
-            custom_field_context = _get_zendesk_custom_field_context(request, learner_data=enterprise_learner_data)
-            custom_fields = _format_zendesk_custom_fields(custom_field_context)
+        help_desk_service = HelpDeskService().get_help_desk_service()
 
-        success = _record_feedback_in_zendesk(
+        success = help_desk_service().record_feedback(
```

Comment:
Where are we passing the `support_email` and `custom_fields` for zendesk?

Reply:
In this commit: https://github.com/proversity-org/edx-platform/commit/347fdb6c822a3fb5a7469d7793891c0d4cd28d6c were removed so, i don't know if we need them. What do you think? @diegomillan